code/index.html

```diff
@@ -3,28 +3,189 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Metal Festival &#x1F918;</title>
+    <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+  <!--HEADER HERO & NAVBAR -->
+
+    <header class="header-hero">
+      <nav class="navbar">
+        <a href="#">ARTISTS</a>
+        <a href="#">TICKETS</a>
+        <a href="#">CAMPING</a>
+        <a href="#">ABOUT</a>
+        <div class="line"></div>
+      </nav>
+      <video
+        class="hero-video"
+        loop
+        autoplay
+        poster="metal-festival.jpg"
+      >
+        <source src="alcest-concert.mp4" type="video/mp4">
+        <source src="alcest-concert.webm" type="video/webm">
+        <source scr="alcest-concert.ogg" type="video/ogg">
+        <p>This browser doesn't support video.........</p>
+      </video>
+    </header>
+    
+    <nav class="navbar-smaller-devices">
+      <a href="#">ARTISTS</a>
+      <a href="#">TICKETS</a>
+      <a href="#">CAMPING</a>
+      <a href="#">ABOUT</a>
+    </nav>
+  
+  <!-- WELCOMTE TITLE -->
+    <h1 class="welcome-title">WELCOME TO METAL FESTIVAL</h1>
+
+  <!-- FORM & ACCORDION -->
+    <main class="form-and-accordion"> 
+
+    <!-- SIGN UP FORM -->
+      <form 
+        class="sign-up-form" 
+        action="https://httpbin.org/anything" 
+        method="POST"
+      >
+        <h2>SIGN UP</h2>
+        <label for="firstName">First name:</label>
+        <br>
+        <br>
```

Comment:
Using `<br>`is considered bad practise - try to use CSS styling to make it look the way you'd like instead!

Reply:
Ah okay :)